rho/utilities.py

```diff
@@ -338,3 +338,17 @@ def get_config_path(filename):
     :returns path to for filename in XDG_CONFIG_HOME associated with rho
     """
     return os.path.join(xdg_config_home, RHO_PATH, filename)
+
+
+def check_path_validity(path_list):
+    """ Given a list of paths it verifies that all paths are valid
+    absolute path inputs for a scoped scan. If they return a list of
```

Comment:
Nit: I think you meant "if not"? Or just change to "returns a list of invalid paths, or []"